src/main/java/uk/co/ramp/covid/simulation/place/Household.java

```diff
@@ -298,4 +267,35 @@ public void dayEnd() {
             isolationTimer--;
         }
     }
+
+    // Household Type management
+    protected int adults = 0;
+    protected int children = 0;
+    protected int pensioners = 0;
+
+    public abstract boolean adultRequired();
```

Comment:
I think a comment here, explaining how these 'required' and 'allowed' methods are used would be useful, or perhaps it would be even better to give them more descriptive names.  At first glance, for example, it seems odd that adults are not allowed in two adult households.

Reply:
Agreed - I'll come up with some better names/documentation.